src/Components/Main.js

```diff
@@ -2,13 +2,37 @@ import React, { Component } from 'react';
 import './Main.css';
 
 class Main extends Component {
+
+  state = {
+    character: { name: 'yo' }
+  }
+
+  move = () => {
+    fetch('//localhost:8080/perform/move', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        direction: 'east'
+      })
+    }).then(res => res.json())
+      .then(character => {
+        this.setState({character})
+      })
+  }
+
   render() {
+    const variable = 520
     return (
       <div className="main-container">
         <div className="stats">Stats</div>
-        <div className="battle">Fight fight fight!</div>
-        <div className="map">Map (x:0, y:0)</div>
-        <div className="unused">Unused</div>
+        <div className="battle">
+          <span>Fight fight fight!</span>
+          <button onClick={e => this.move()} >Move</button>
```

Comment:
Usually good to name this "event" instead of "e". Javascript already lacks types, so it can help (especially for newer people like Adam)